src/Field.tsx

```diff
@@ -83,110 +59,108 @@ export interface FieldConfig {
 
 export type FieldAttributes<T> = GenericFieldHTMLAttributes & FieldConfig & T;
 
-/**
- * Custom Field component for quickly hooking into Formik
- * context and wiring up forms.
- */
-class FieldInner<Values = {}, Props = {}> extends React.Component<
-  FieldAttributes<Props> & { formik: FormikContext<Values> },
-  {}
-> {
-  constructor(
-    props: FieldAttributes<Props> & { formik: FormikContext<Values> }
-  ) {
-    super(props);
-    const { render, children, component } = props;
-    warning(
-      !(component && render),
-      'You should not use <Field component> and <Field render> in the same <Field> component; <Field component> will be ignored'
-    );
-
-    warning(
-      !(component && children && isFunction(children)),
-      'You should not use <Field component> and <Field children> as a function in the same <Field> component; <Field component> will be ignored.'
-    );
+export function useField(name: string, type?: string) {
+  const formik = useFormikContext();
 
+  if (process.env.NODE_ENV !== 'production') {
     warning(
-      !(render && children && !isEmptyChildren(children)),
-      'You should not use <Field render> and <Field children> in the same <Field> component; <Field children> will be ignored'
+      !formik,
```

Comment:
it should be `formik` see:
https://github.com/alexreardon/tiny-warning/blob/dc65095d4e27290869781c778950ed82a91de866/src/index.js#L9